nyc/bills.py

```diff
@@ -161,133 +161,156 @@ def sponsorships(self, matter_id):
+        if text:
+            if text['MatterTextPlain']:
+                bill.extras['plain_text'] = text['MatterTextPlain'].replace(u'\u0000', '')
 
-            for topic in self.topics(matter_id) :
-                bill.add_subject(topic['MatterIndexName'].strip())
+            if text['MatterTextRtf']:
+                bill.extras['rtf_text'] = text['MatterTextRtf'].replace(u'\u0000', '')
 
-            for relation in self.relations(matter_id):
-                try:
-                    related_bill = self.endpoint('/matters/{0}', relation['MatterRelationMatterId'])
-                except scrapelib.HTTPError:
-                    continue
-                else:
-                    date = related_bill['MatterIntroDate']
-                    related_bill_session = self.session(self.toTime(date))
-                    identifier = related_bill['MatterFile']
-                    bill.add_related_bill(identifier=identifier,
-                                          legislative_session=related_bill_session,
-                                          relation_type='companion')
+        return bill
 
-            try:
-                text = self.text(matter_id)
-            except KeyError:
-                version_errors.append(legistar_web)
-                continue
 
+    def get_vote_event(self, bill, act, vote):
+        '''Make VoteEvent object from given Bill, action, and vote.'''
+        result, votes = vote
+
+        if result:
+            organization = json.loads(act['organization_id'].lstrip('~'))
+            vote_event = VoteEvent(legislative_session=bill.legislative_session,
+                                   motion_text=act['description'],
+                                   organization=organization,
+                                   classification=None,
+                                   start_date=act['date'],
+                                   result=result,
+                                   bill=bill)
 
-            if text:
-                if text['MatterTextPlain']:
-                    bill.extras['plain_text'] = text['MatterTextPlain'].replace(u'\u0000', '')
+            legistar_web, legistar_api = [src['url'] for src in bill.sources]
 
-                if text['MatterTextRtf']:
-                    bill.extras['rtf_text'] = text['MatterTextRtf'].replace(u'\u0000', '')
+            vote_event.add_source(legistar_web)
+            vote_event.add_source(legistar_api + '/histories')
 
+            for vote in votes:
+                raw_option = vote['VoteValueName'].lower()
 
-            for action, vote in self.actions(matter_id):
-                act = bill.add_action(action['action_description'],
-                                      action['action_date'],
-                                      organization={'name': action['responsible_org']},
-                                      classification=action['classification'])
+                if raw_option == 'suspended':
+                    return None
```

Comment:
this should be a `continue` I'm pretty sure.

Reply:
great catch!